README.md

```diff
@@ -137,6 +138,7 @@ Array.from(tuple).map(x => new MyClass(x))
 assert((#{} with .a = 1, .b = 2) === #{ a: 1, b: 2 });
 assert((#[ {} ] with [0].a = 1) === #[ { a: 1 } ]);
 assert((x = 0, #[ {} ] with [x].a = 1) === #[ { a: 1 } ]);
+assert.equal(#{ a: 1 }, Record.from(JSON.stringify(#{ a: 1 })));
```

Comment:
should this be:

```
assert(#{ a: 1 }, Record.from(JSON.parse(JSON.stringify(#{ a: 1 }))));
```

i.e. the string should be converted to an object first via `JSON.parse`